src/breakpoints/index.ts

```diff
@@ -22,16 +26,31 @@ export class Breakpoints extends Panel {
       'deactivate',
       new ToolbarButton({
         iconClassName: 'jp-DebuggerDeactivateIcon',
+        tooltip: `${this.isAllAcitve ? 'Deactivate' : 'Activate'} Breakpoints`,
         onClick: () => {
-          console.log('`deactivate` was clicked');
+          this.isAllAcitve = !this.isAllAcitve;
```

Comment:
Little typo spotted:

```suggestion
          this.isAllActive = !this.isAllActive;
```